### ISSUE TITLE
Quray about NI for SAM

### ISSUE THREAD

**Issue by alicejiang7888** [u6630774/TASS#1]
Thanks for your wonderful work, I saw your results of using NI attack SAM, and I'm curious about what loss function you used to optimize adversarial samples for SAM, still using CrossEntropyLoss or other loss function? As I know SAM dataset did not have a label. I did not find this part in your code, Thanks a lot~
